test/close.test.js

```diff
@@ -328,6 +328,20 @@ test('close without a cb', (t) => {
   app.close()
 })
 
+test('close without an instance', (t) => {
+  t.plan(2)
+
+  const app = boot()
+
+  app.onClose((done) => {
+    t.ok('called')
+  })
+
+  app.close(() => {
+    t.pass('closed')
```

Comment:
you might want to:

1. use a `function ( ) {}` so you have access to the `arguments` object
2. place an assertion that checks that `arguments.length === 0`

Reply:
Looks like length is always 1 as `{0 : null}`  so I can check like:

```js
    t.is(arguments[0], null)
    t.is(arguments[1], undefined)
```

Is that alright?

Reply:
then there is possibly something off with your code, as https://github.com/mcollina/avvio/pull/76/files#diff-fe60a8aed4b3103661f9d88361346b0dR353 should be triggered.